hrs-portlets-webapp/src/main/java/edu/wisc/portlet/hrs/web/listoflinks/Link.java

```diff
@@ -10,6 +10,12 @@
   private String target;
   private String icon;
 
+  public Link(String title, String href, String target, String icon) {
```

Comment:
This removed the implied no-arg constructor, restored immediately following this merge in 8fe93d8e8fe84ea2167578051bffa5bbaced0fce .